microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

```diff
@@ -48,7 +48,7 @@ For a webpage, you can use the [prefers-color-scheme][MDNPrefersColorScheme] CSS
 
```

Comment:
At bottom, change:
View and change your CSS like any other web page
to:
Modify your CSS and view the rendered result the same way as for any other web page.